central/administration/events/service/singleton.go

```diff
@@ -0,0 +1,24 @@
+package service
+
+import (
+	"github.com/stackrox/rox/central/administration/events/datastore"
+	"github.com/stackrox/rox/pkg/features"
+	"github.com/stackrox/rox/pkg/sync"
+)
+
+var (
+	once sync.Once
+
+	svc Service
+)
+
+// Singleton provides the instance of the Service interface to register.
+func Singleton() Service {
```

Comment:
Also, this is probably a good time to add this to the grpc server so it shows up when you run the product with the flag on :)